dev/components/stocks.js

```diff
@@ -1,36 +1,47 @@
 import $ from 'jquery';
-
+import _ from 'lodash';
+import store from 'store2';
+import StockPopUp from './stock-popup.js';
 
 class Stocks {
   constructor(container) {
-
     // REGISTER ELEMENTS
     this.$container = container;
+    this.graph;
+    this.popup;
 
     this.render();
-
     this.$stockListContainer = $('#stocks-list');
-
-    // INITIALIZE FETCH OF STOCKS
-    this.getStocks();
+    this.count = 1;
+
+    // check if local storage exist
+    if (store.get(`stocks${this.count}`)) {
```

Comment:
This is highly prone to errors, if they 'stocks' item doesn't exist then this may break. consider this instead: 
```
const stocks = store.get('stocks') || []; 

if(stocks.length) { ...
```